src/accounts/index.js

```diff
@@ -0,0 +1,110 @@
+import _ from 'lodash';
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+import db from '../../models';
+import responseHelper from '../helpers/response';
+
+dotenv.config();
+
+class AccountController {
```

Comment:
This class is doing so many things it's hard to keep track. Could you break this down so we can have this class do just one thing? @albert85

Reply:
ok on it